great-idea-website/css/index.css

```diff
@@ -63,4 +63,153 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body {
+	width:900px;
+	margin: auto;
+}
+nav {
+	width: 100%;
+	margin-top: 10px;
+	position: relative;
+	display: inline-block;
+}
+
+nav .nav-items {
+	width: 65%;
+	display: inline-block;
+	position: relative;
+	text-align: center;
+	vertical-align: middle;
+	margin-bottom: 16px;
+}
+
+nav .nav-items a {
+	display: inline-block;
+	position: relative;
+	padding-left: 10px;
+	padding-right: 30px;
+	font-size: 16px;
+	text-decoration: none;
+	color: grey;
+}
+
```

Comment:
Navigation looks great. One thing, the logo in the design file is a little more flush with the edge of the container. Looks like you have a pretty big margin around that logo so it should be an easy fix.